cypress/e2e/tests/migration/reports/filter.test.ts

```diff
@@ -112,8 +112,13 @@ describe(["@tier2"], "Reports filter validations", () => {
         clickByText(button, clearAllFilters);
 
         //close Row Details of Identified risks
-        click(closeRowIdentifiedRisk);
-
+        cy.contains("h3", "Identified risks")
+            .closest("div.pf-v5-c-card") // Find the closest div with the given class
+            .find("div.pf-v5-c-card__header") // Within that div, find the header div
+            .within(() => {
+                cy.get("button") // Get the button within the header
```

Comment:
These comments are not required.

//Get the button and // Click the button 
// Find the closest div with the given class

// Within that div, find the header div

Reply:
Done